features/puppet/defaults.pan

```diff
@@ -0,0 +1,14 @@
+template features/puppet/defaults;
+
+include 'quattor/functions/package';
+
+include 'components/puppet/config';
+
+variable PUPPET_4_INCLUDE = is_defined(PUPPET_VERSION) &&
```

Comment:
Would it be acceptable to provide a default Puppet version to avoid having to check for it being defined?

For example:
```pan
variable PUPPET_VERSION ?= '3.0.0';
```

Reply:
Ok I see your comment, nevermind.